curriculum/test/test-challenges.js

```diff
@@ -253,6 +251,44 @@ function populateTestsForLang({ lang, challenges, meta }) {
   const challengeTitles = new ChallengeTitles();
   const validateChallenge = challengeSchemaValidator();
 
+  describe('Assert meta order', function () {
+    /** This array can be used to skip a superblock - we'll use this
+     * when we are working on the new project-based curriculum for
+     * a superblock (because keeping those challenges in order is
+     * tricky and needs cleaning up before deploying).
+     */
+    const superBlocksUnderDevelopment = ['responsive-web-design'];
+    const superBlocks = new Set([
+      ...Object.values(meta)
+        .map(el => el.superBlock)
+        .filter(el => !superBlocksUnderDevelopment.includes(el))
+    ]);
+    superBlocks.forEach(superBlock => {
+      const filteredMeta = Object.values(meta)
+        /**
+         * Exclude any meta which doesn't have a superOrder, as these shouldn't
+         * appear on the learn map and thus don't need to be validated.
+         */
+        .filter(el => el.superBlock === superBlock && el.superOrder)
+        .sort((a, b) => a.order - b.order);
```

Comment:
@ojeytonwilliams my thought process here is that if `superOrder` is not present (I removed them from the certification metas), then it's not going to be rendered on the map and we don't care about the order. By excluding those in the filter, the test won't run as there won't be a `filteredmeta`. 

Is my understanding here correct?

Reply:
I think so, Nick.  From what I can see, `superOrder` really only matters for the total challenge order and for ordering superblocks on the map.

The certificates aren't challenges and shouldn't appear on the map, so I can't think of a good reason for them to have a `superOrder`.

Reply:
Alright, this should all be taken care of then. 🙂